llvm/utils/git/github-automation.py

```diff
@@ -24,12 +24,12 @@
 
 This issue may be a good introductory issue for people new to working on LLVM. If you would like to work on this issue, your first steps are:
 
-1. In the comments of the issue, request for it to be assigned to you.
+1. In the comments of this issue, request for it to be assigned to you, or just create a [pull request](https://github.com/llvm/llvm-project/pulls) after following the steps below. [Mention](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue) this issue in the description of the pull request.
```

Comment:
As I said earlier, I think assignment should be mandatory. The text should also make it clear that you should communicate with an existing assignee before creating your own PR, give a reasonable amount of time (approx 7 days) for them to respond, and then update the assignment if you agree to take over the ticket.

Reply:
Updated. Okay for you @RKSimon?

Reply:
Fix the numbered list?

Reply:
What do you mean? This text will be rendered by GitHub as Markdown. As such, numbered lists items can all start with `1.` to avoid adjusting the numbers later when a new item is added.

Reply:
OK - it was manually done beforehand, if you're confident it will work that's fine :)